system_metrics_collector/src/topic_statistics_collector/topic_statistics_node.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "libstatistics_collector/topic_statistics_collector/constants.hpp"
+#include "subscriber_topic_statistics.hpp"
+#include "system_metrics_collector/msg/dummy_message.hpp"
+
+/**
+ * An entry point that starts the topic statistics collector node.
+ */
+
+int main(int argc, char ** argv)
+{
+  namespace constants =
+    libstatistics_collector::topic_statistics_collector::topic_statistics_constants;
+
+  rclcpp::init(argc, argv);
+
+  const auto options = rclcpp::NodeOptions().append_parameter_override(
+    constants::kCollectStatsTopicNameParam,
+    "dummy_data");
```

Comment:
is this correct? I think this needs a comment

Reply:
If this is for the e2e test then we should probably document as such in the main. Otherwise I'm confused if this is the generic entry point (using dummy data).

Reply:
This is for the e2e test. I'll add a comment around that.